Submitted By: DJ Lucas <dj AT linuxfromscratch DOT org>
Date: 2006-06-25
Initial Package Version: 1.4.2
Upstream Status: Not applicable
Origin: Previous fixed paths patch
Description: Instead of fixing the path where binaries are located, use the
             binaries in the user's path. Updated for 6u1

--- jdk-build-orig/j2se/make/common/shared/Defs-utils.gmk	2007-02-10 15:22:33.000000000 -0600
+++ jdk-build/j2se/make/common/shared/Defs-utils.gmk	2007-02-09 17:04:06.000000000 -0600
@@ -52,73 +52,73 @@
 endif
 
 # Utilities
-ADB            = $(UTILS_COMMAND_PATH)adb
-AR             = $(UTILS_CCS_BIN_PATH)ar
-AS             = $(UTILS_CCS_BIN_PATH)as
-BASENAME       = $(UTILS_COMMAND_PATH)basename
-CAT            = $(UTILS_COMMAND_PATH)cat
-CHMOD          = $(UTILS_COMMAND_PATH)chmod
-CMP            = $(UTILS_USR_BIN_PATH)cmp
-COMM           = $(UTILS_USR_BIN_PATH)comm
-COMPRESS       = $(UTILS_USR_BIN_PATH)compress
-CP             = $(UTILS_COMMAND_PATH)cp
-CPIO           = $(UTILS_COMMAND_PATH)cpio
-CUT            = $(UTILS_USR_BIN_PATH)cut
-DATE           = $(UTILS_COMMAND_PATH)date
-DF             = $(UTILS_COMMAND_PATH)df
-DIFF           = $(UTILS_USR_BIN_PATH)diff
-DIRNAME        = $(UTILS_USR_BIN_PATH)dirname
-ECHO           = $(UTILS_COMMAND_PATH)echo
-EGREP          = $(UTILS_COMMAND_PATH)egrep
-EXPR           = $(UTILS_USR_BIN_PATH)expr
-FILE           = $(UTILS_USR_BIN_PATH)file
-FIND           = $(UTILS_USR_BIN_PATH)find
-FMT            = $(UTILS_COMMAND_PATH)fmt
-GDB            = $(UTILS_USR_BIN_PATH)gdb
-GREP           = $(UTILS_COMMAND_PATH)grep
-GUNZIP         = $(UTILS_COMMAND_PATH)gunzip
-HEAD           = $(UTILS_USR_BIN_PATH)head
-ID             = $(UTILS_COMMAND_PATH)id
-ISAINFO        = $(UTILS_COMMAND_PATH)isainfo
-KSH	       = $(UTILS_COMMAND_PATH)ksh
-LD             = $(UTILS_CCS_BIN_PATH)ld
-LEX            = $(UTILS_CCS_BIN_PATH)lex
-LN             = $(UTILS_COMMAND_PATH)ln
-LS             = $(UTILS_COMMAND_PATH)ls
-MCS            = $(UTILS_CCS_BIN_PATH)mcs
-M4             = $(UTILS_CCS_BIN_PATH)m4
-MKDIR          = $(UTILS_COMMAND_PATH)mkdir
-MKSINFO        = $(UTILS_COMMAND_PATH)mksinfo
-MSGFMT         = $(UTILS_USR_BIN_PATH)msgfmt
-MV             = $(UTILS_COMMAND_PATH)mv
-NAWK           = $(UTILS_USR_BIN_PATH)nawk
-NM             = $(UTILS_CCS_BIN_PATH)nm
-PKGMK	       = $(UTILS_COMMAND_PATH)pkgmk
-PRINTF         = $(UTILS_USR_BIN_PATH)printf
-PWD            = $(UTILS_COMMAND_PATH)pwd
-RC             = $(UTILS_COMMAND_PATH)rc
-RMDIR          = $(UTILS_COMMAND_PATH)rmdir
-RPM            = $(UTILS_COMMAND_PATH)rpm
-SCCS           = $(UTILS_CCS_BIN_PATH)sccs
-SED            = $(UTILS_COMMAND_PATH)sed
-SH             = $(UTILS_COMMAND_PATH)sh
-SHOWREV        = $(UTILS_USR_BIN_PATH)showrev
-SORT           = $(UTILS_COMMAND_PATH)sort
-STRIP          = $(UTILS_CCS_BIN_PATH)strip
-TAIL           = $(UTILS_USR_BIN_PATH)tail
-TAR            = $(UTILS_COMMAND_PATH)tar
-TEST           = $(UTILS_USR_BIN_PATH)test
-TOUCH          = $(UTILS_COMMAND_PATH)touch
-TR             = $(UTILS_USR_BIN_PATH)tr
-TRUE           = $(UTILS_COMMAND_PATH)true
-UNAME          = $(UTILS_COMMAND_PATH)uname
-UNIQ           = $(UTILS_USR_BIN_PATH)uniq
-UNZIP          = $(UTILS_USR_BIN_PATH)unzip
-UNZIPSFX       = $(UTILS_DEVTOOL_PATH)unzipsfx
-WC             = $(UTILS_USR_BIN_PATH)wc
-WHICH          = $(UTILS_USR_BIN_PATH)which
-YACC           = $(UTILS_CCS_BIN_PATH)yacc
-ZIPEXE         = $(UTILS_DEVTOOL_PATH)zip
+ADB            = adb
+AR             = ar
+AS             = as
+BASENAME       = basename
+CAT            = cat
+CHMOD          = chmod
+CMP            = cmp
+COMM           = comm
+COMPRESS       = compress
+CP             = cp
+CPIO           = cpio
+CUT            = cut
+DATE           = date
+DF             = df
+DIFF           = diff
+DIRNAME        = dirname
+ECHO           = echo
+EGREP          = egrep
+EXPR           = expr
+FILE           = file
+FIND           = find
+FMT            = fmt
+GDB            = gdb
+GREP           = grep
+GUNZIP         = gunzip
+HEAD           = head
+ID             = id
+ISAINFO        = isainfo
+KSH	       = ksh
+LD             = ld
+LEX            = lex
+LN             = ln
+LS             = ls
+MCS            = mcs
+M4             = m4
+MKDIR          = mkdir
+MKSINFO        = mksinfo
+MSGFMT         = msgfmt
+MV             = mv
+NAWK           = nawk
+NM             = nm
+PKGMK	       = pkgmk
+PRINTF         = printf
+PWD            = pwd
+RC             = rc
+RMDIR          = rmdir
+RPM            = rpm
+SCCS           = sccs
+SED            = sed
+SH             = sh
+SHOWREV        = showrev
+SORT           = sort
+STRIP          = strip
+TAIL           = tail
+TAR            = tar
+TEST           = test
+TOUCH          = touch
+TR             = tr
+TRUE           = true
+UNAME          = uname
+UNIQ           = uniq
+UNZIP          = unzip
+UNZIPSFX       = unzipsfx
+WC             = wc
+WHICH          = which
+YACC           = yacc
+ZIPEXE         = zip
 
 # Special cases
 #RM   is defined by GNU Make as 'rm -f'
@@ -144,22 +144,21 @@
 ifeq ($(PLATFORM),linux)
   # A few redefines
   # Allows for builds on Debian GNU Linux, basename is in a different place
-  BASENAME=$(firstword $(wildcard $(UTILS_COMMAND_PATH)basename) \
-                       $(wildcard $(UTILS_USR_BIN_PATH)basename))
+  BASENAME = basename
 
-  NAWK           = $(USRBIN_PATH)gawk
+  NAWK           = gawk
   # Intrinsic unix command, with backslash-escaped character interpretation
   ECHO           = echo -e
   # These are really in UTILS_USR_BIN_PATH on Linux (only sccs is not)
-  AR             = $(UTILS_USR_BIN_PATH)ar
-  AS             = $(UTILS_USR_BIN_PATH)as
-  LD             = $(UTILS_USR_BIN_PATH)ld
-  LEX            = $(UTILS_USR_BIN_PATH)lex
-  MCS            = $(UTILS_USR_BIN_PATH)mcs
-  M4             = $(UTILS_USR_BIN_PATH)m4
-  NM             = $(UTILS_USR_BIN_PATH)nm
-  STRIP          = $(UTILS_USR_BIN_PATH)strip
-  YACC           = $(UTILS_USR_BIN_PATH)yacc
+  AR             = ar
+  AS             = as
+  LD             = ld
+  LEX            = lex
+  MCS            = mcs
+  M4             = m4
+  NM             = nm
+  STRIP          = strip
+  YACC           = yacc
 endif
 
 # Solaris specific
--- jdk-build-orig/deploy/make/common/Defs-linux.gmk	2007-02-10 18:09:25.000000000 -0600
+++ jdk-build/deploy/make/common/Defs-linux.gmk	2007-02-10 18:08:02.000000000 -0600
@@ -9,7 +9,7 @@
 
 BUNDLE_SUFFIX  = .zip
 CC	       = $(COMPILER_PATH)gcc
-HEAD           = $(USRBIN_PATH)head
+HEAD           = head
 CLASSPATH_SEP  = $(CLASSPATH_SEPARATOR)
 JUMP_EXE       = $(BIN_DIR)/javaws
 PATH_SEP       = /
